.github/workflows/build_and_test_workflow.yml

```diff
@@ -90,7 +95,7 @@ jobs:
 
       # github virtual env is the latest chrome
       - name: Setup chromedriver
-        run: yarn add --dev chromedriver@100.0.0
+        run: yarn add --dev chromedriver@99.0.0
```

Comment:
Why is v99 preferable here?

Reply:
Updated the comment. The image we use has Chrome 99.